libs/@blockprotocol/graph/src/graph-module.json

```diff
@@ -181,512 +170,10 @@
-            "description": "The aggregation operation to apply",
-            "$ref": "https://blockprotocol.org/types/modules/graph/aggregation-operation"
-          }
-        },
-        "required": ["operation"]
-      },
-      "errorCodes": []
-    },
-    {
-      "messageName": "aggregateEntityTypesResponse",
-      "description": "The response to a request to retrieve an aggregation of entity types",
-      "source": "embedder",
-      "respondedToBy": null,
-      "sentOnInitialization": false,
-      "data": {
-        "type": "object",
-        "properties": {
-          "operation": {
-            "description": "The aggregation operation that was applied to generate the results, including any defaults set by the application.",
-            "$ref": "https://blockprotocol.org/types/modules/graph/aggregation-operation"
-          },
-          "results": {
-            "description": "The entity types returned by the aggregation operation",
-            "type": "array",
-            "items": {
-              "$ref": "https://blockprotocol.org/types/modules/graph/entity-type"
-            }
-          }
-        },
-        "required": ["operation", "results"]
-      },
-      "errorCodes": ["FORBIDDEN", "INVALID_INPUT"]
-    },
-    {
-      "messageName": "createLink",
-      "description": "Request to create a link, expecting the created link in response.",
-      "source": "block",
-      "respondedToBy": "createLinkResponse",
-      "sentOnInitialization": false,
-      "data": {
-        "type": "object",
-        "properties": {
-          "sourceEntityId": {
-            "description": "The entityId of the source entity for the link",
-            "type": "string"
-          },
-          "destinationEntityId": {
-            "description": "The entityId of the destination entity for the link",
-            "type": "string"
-          },
-          "index": {
-            "description": "The position of this link in an ordered list of links, where ordering is important. An index of 0 represents the first position in the list.",
-            "type": "number"
-          },
-          "path": {
-            "description": "The path or field on the entity where this link is made (e.g. 'friend', 'employer')",
-            "type": "string"
-          }
-        },
-        "required": ["sourceEntityId", "destinationEntityId", "path"]
-      },
-      "errorCodes": []
-    },
-    {
-      "messageName": "createLinkResponse",
-      "description": "The response to a request to create a link",
-      "source": "embedder",
-      "respondedToBy": null,
-      "sentOnInitialization": false,
-      "data": {
-        "type": "object",
-        "properties": {
-          "link": {
-            "description": "The created link",
-            "$ref": "https://blockprotocol.org/types/modules/graph/link"
-          }
-        },
-        "required": ["link"]
+        "$ref": "https://blockprotocol.org/types/modules/graph/0.3/schema/subgraph"
```

Comment:
Does this match the type in the package?

It looks like it returns
```ts
{
  data:  {
    results: Subgraph<EntityRootType>;
    operation: AggregateOperationInput
  }
}
```

whereas here you're just returning 
```ts
{
  data: Subgraph<EntityRootType>
}
```

Reply:
Good spot, I fixed it in [Fix aggregation operation](https://github.com/blockprotocol/blockprotocol/pull/1034/commits/61e6b7a5d4ea03cba90759ae2f0128d9598e4396)